config/locales/login_cancel/en.yml

```
@@ -0,0 +1,11 @@
+---
+en:
+  login_cancel:
+    account_page: account page
+    bullet1: You won’t be able to use %{app_name} to access %{sp_name} until you
```

In the design, `%{sp_name}` is shown as bolded.

```suggestion
    bullet1_html: You won’t be able to use %{app_name} to access <strong>%{sp_name}</strong> until you
```